test.toml

```diff
@@ -16,8 +16,10 @@ database_url = 'sqlite:///ursabot.sqlite'
 name = 'local1'
 arch = 'amd64'
 docker = { host = 'unix://var/run/docker.sock' }
+hostconfig = { network_mode = 'host', cpuset_cpus = '0,1' }
```

Comment:
Move these config keys under docker, because We'll have non-docker workers as well.

Reply:
Done